server/athenian/api/preloading/__main__.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python3
+import argparse
+import asyncio
+import logging
+
+from athenian.api import metadata
+from athenian.api.async_utils import gather
+from athenian.api.db import check_schema_versions, compose_db_options, ParallelDatabase
+from athenian.api.preloading.cache import MemoryCachePreloader
+
+
+def parse_args() -> argparse.Namespace:
+    """Parse the command line and return the parsed arguments."""
+    parser = argparse.ArgumentParser(metadata.__package__)
+
+    parser.add_argument("--metadata-db",
+                        default="postgresql://postgres:postgres@0.0.0.0:5432/metadata",
+                        help="Metadata (GitHub, JIRA, etc.) DB connection string in SQLAlchemy "
+                             "format. This DB is readonly.")
+    parser.add_argument("--state-db",
+                        default="postgresql://postgres:postgres@0.0.0.0:5432/state",
+                        help="Server state (user settings, teams, etc.) DB connection string in "
+                             "SQLAlchemy format. This DB is read/write.")
+    parser.add_argument("--precomputed-db",
+                        default="postgresql://postgres:postgres@0.0.0.0:5432/precomputed",
+                        help="Precomputed objects augmenting the metadata DB and reducing "
+                             "the amount of online work. DB connection string in SQLAlchemy "
+                             "format. This DB is read/write.")
+    parser.add_argument("--persistentdata-db",
+                        default="postgresql://postgres:postgres@0.0.0.0:5432/persistentdata",
+                        help="Pushed and pulled source data that Athenian owns. DB connection "
+                             "string in SQLAlchemy format. This DB is read/write.")
+    parser.add_argument("--detailed", required=False, action="store_true",
+                        help="Whether to also show the memory usage of each column.")
+
+    return parser.parse_args()
+
+
+def main():
+    """Print memory usage for preloaded DataFrames."""
+    args = parse_args()
+    log = logging.getLogger(metadata.__package__)
+    if not check_schema_versions(args.metadata_db,
+                                 args.state_db,
+                                 args.precomputed_db,
+                                 args.persistentdata_db,
+                                 log):
+        return None
+
+    db_conns = {
+        "sdb": args.state_db,
+        "mdb": args.metadata_db,
+        "pdb": args.precomputed_db,
+        "rdb": args.persistentdata_db,
+    }
+    db_options = compose_db_options(args.metadata_db,
+                                    args.state_db,
+                                    args.precomputed_db,
+                                    args.persistentdata_db)
+
+    asyncio.run(_main(db_conns, db_options, args.detailed))
```

Comment:
```suggestion
    return asyncio.run(_main(db_conns, db_options, args.detailed))
```
It returns the `_main`'s result.